src/main/resources/assets/farmersdelight/lang/ru_ru.json

```diff
@@ -206,12 +206,12 @@
   "effect.farmersdelight.comfort": "Комфорт",
 
   "effect.farmersdelight.nourishment.0.desc": "Предотвращает потерю голода, за исключением случаев использования насыщения для лечения.",
-  "effect.farmersdelight.comfort.0.desc": "Делает вас невосприимчивым к Голоду, Медлительности и Слабости.",
+  "effect.farmersdelight.comfort.0.desc": "Делает вас невосприимчивым к голоду, медлительности и слабости.",
```

Comment:
This description is outdated. Comfort has a different effect now:
"Provides natural regeneration, regardless of how hungry you are."

Reply:
Done.